client/my-sites/checkout/composite-checkout/lib/get-jetpack-product-features.ts

```diff
@@ -1,20 +1,179 @@
-import { isJetpackBackupT1Slug } from '@automattic/calypso-products';
+import {
+	isJetpackAntiSpamSlug,
+	isJetpackBackupT1Slug,
+	isJetpackBoostSlug,
+	isJetpackCompleteSlug,
+	isJetpackScanSlug,
+	isJetpackSearchSlug,
+	isJetpackSecurityT1Slug,
+	isJetpackSocialBasicSlug,
+	isJetpackSocialAdvancedSlug,
+	isJetpackVideoPressSlug,
+} from '@automattic/calypso-products';
 import { useTranslate } from 'i18n-calypso';
 import type { WithSnakeCaseSlug } from '@automattic/calypso-products';
 
+type featureString =
+	| 'anti-spam'
+	| 'backup-t1'
+	| 'boost'
+	| 'scan'
+	| 'search'
+	| 'social-basic'
+	| 'social-advanced'
+	| 'support'
+	| 'videopress'
+	| 'complete';
+
+function getFeatureStrings(
+	feature: featureString,
+	translate: ReturnType< typeof useTranslate >
+): string[] {
+	switch ( feature ) {
+		case 'anti-spam':
+			return [
+				translate( 'Comment and form spam protection' ),
+				translate( '10K API calls per month' ),
+				translate( 'Akismet technology' ),
+				translate( 'Flexible API' ),
+			];
+		case 'backup-t1':
+			return [
+				translate( 'Real-time cloud backups' ),
+				translate( '10GB of backup storage' ),
+				translate( '30-day archive & activity log' ),
+				translate( 'One-click restores' ),
+			];
+		case 'boost':
+			return [
+				translate( 'Automated critical CSS' ),
+				translate( 'Site performance scores' ),
+				translate( 'One-click optimization' ),
+				translate( 'Deferred non-essential JavaScript' ),
+				translate( 'Optimized CSS loading' ),
+				translate( 'Lazy image loading' ),
+			];
+		case 'complete':
+			return [
+				translate( 'Akismet Anti-spam' ),
+				translate( 'VaultPress Backup' ),
+				translate( 'Site Search' ),
+				translate( 'Social' ),
+				translate( 'Scan' ),
+				translate( 'VideoPress' ),
+				translate( 'Boost' ),
+				translate( 'CRM Entrepreneur' ),
+			];
+		case 'scan':
+			return [
+				translate( 'Website firewall (WAF beta)' ),
+				translate( 'Automated daily scanning' ),
+				translate( 'One-click fixes for most issues' ),
+				translate( 'Instant email threat notifications' ),
+			];
+		case 'search':
+			return [
+				translate( 'Instant search, filtering, and indexing' ),
+				translate( 'Highly relevant search results' ),
+				translate( 'Quick and accurate spelling correction' ),
+				translate( 'Support for 38 languages' ),
+			];
+		case 'social-advanced':
+			return [ translate( 'Engagement Optimizer' ) ];
```

Comment:
I wasn't sure whether or not to add any of the "Coming soon" features 🤔

Reply:
![image](https://user-images.githubusercontent.com/65001528/220434690-f4bd472e-f25c-4f84-bdc5-9078a8ab6404.png)

Reply:
It's just my humble opinion, but I would say not to include them. My thought is that if the user is already at checkout they want to purchase from us, and I suspect these might be a deterrent rather than an encouragement. 

I suspect that adding the `Coming soon` options might make people pause checkout or think they should come back when those features are available.

Testing the behavior would be the only way to confirm either way.